packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1468,24 +1415,53 @@ export async function runTestForSeed<
 				),
 		  );
 	const summarizerClient = initialClient;
+	const handles = Array.from({ length: 5 }).map(() => uuid());
+	let handleGenerated = false;
 	const initialState: DDSFuzzTestState<TChannelFactory> = {
 		clients,
 		summarizerClient,
 		containerRuntimeFactory,
-		random,
+		random: {
+			...random,
+			handle: () => {
+				if (options.handleGenerationDisabled !== true) {
```

Comment:
i don't think this is necessary, and it makes the typing worse by forcing handle to also support undefined. i don't think its a problem if handles are disabled but someone creates one anyway. the disable is more about disabling the check that handles are created.